config/src/test/java/org/springframework/security/config/annotation/method/configuration/PrePostMethodSecurityConfigurationTests.java

```diff
@@ -627,4 +647,17 @@ Authz authz() {
 
 	}
 
+	@Configuration
+	@EnableMethodSecurity(jsr250Enabled = true, securedEnabled = true)
+	static class RoleHierarchyConfig {
+
+		@Bean
+		RoleHierarchy roleHierarchy() {
```

Comment:
Will you please make this method `static`? Such ensures that it is available at the time the `static` method interceptor beans are published. It also ensures that the interceptor beans are available for bean post-processing.